backend/src/users/users.service.ts

```diff
@@ -112,8 +112,8 @@ export class UsersService {
     const userToCreate = {
       ...user,
       password: user.password ? bcrypt.hashSync(user.password, 10) : undefined,
-      id: organizationId,
       createdAt: new Date(),
+      organizationId,
```

Comment:
The fix doesn't seem to work as we don't have a `organizationId` field on our entity.